### DISCUSSION TITLE
Switching on cameras while the Home Assistant is running

### DISCUSSION THREAD

**Discussion by riwich** [fwestenberg/reolink_dev#509] [Q&A]
Am I perhaps not understanding something correctly? 
My configuration includes a "E1 Zoom" and a "C1 Pro". However, the two cameras are not permanently switched on. Now, when I switch on the cameras, they are not recognised by the integration, even though they worked perfectly before. It only works if I restart Home Assistant when the cameras are switched on, or if I remove the cameras in the integration and integrate them again. 
Not exactly ideal... 
Am I doing something wrong?

**Comment by conma293** [comment]
Hi there, off topic but would you recommend a E1 or C1 ? which is better for HA integration... also, is the siren available to trigger in HA?

**Comment by LeidenSpain** [comment]
Hello,

Can you confirm that the C1 Pro is compatible? I've seen some comments that said no.

Regards,
